homeassistant/components/tuya/light.py

```diff
@@ -123,6 +123,11 @@ class TuyaLightEntityDescription(LightEntityDescription):
             color_temp=DPCode.TEMP_VALUE,
             color_data=DPCode.COLOUR_DATA,
         ),
+        # Some ceiling fan lights use LIGHT for DPCode instead of SWITCH_LED
+        TuyaLightEntityDescription(
```

Comment:
Are there not more properties? Like brightness controls, color or color temperature controls?

Reply:
No other properties, it just turns the light circuit of the ceiling fan on and off.